bench/performances.js

```diff
@@ -116,12 +116,12 @@ async function serverStart() {
 
   configBuilder.set('store:name', 'local_store');
 
-  configBuilder.set('mapStyle:poiMapUrl', `["http://localhost:${PORT}/fake_pbf/{z}/{x}/{y}.pbf"]`);
-  configBuilder.set('mapStyle:baseMapUrl', `["http://localhost:${PORT}/fake_pbf/{z}/{x}/{y}.pbf"]`);
   configBuilder.set('performance:enabled', true);
 
 
   const config = configBuilder.get();
```

Comment:
As no valid is url is defined in the default configuration, this would fail before the values could be overidden.
```suggestion
  const config = configBuilder.get_without_check();
```